cmd/main.go

```diff
@@ -87,7 +87,7 @@ func main() {
 		syscall.SIGINT,
 		syscall.SIGTERM,
 		syscall.SIGQUIT)
-	fmt.Printf("Got signal %d to exit.\n", <-sc)
+	fmt.Fprintf(os.Stderr, "Got signal %d to exit.\n", <-sc)
```

Comment:
May log the quit signal in stdout?

Reply:
I log all messages to stderr and output generated statements to stdout for users to redirect statements.